easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.2.3-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'AUGUSTUS'
+version = '3.2.3'
+versionsuffix = '-Python-2.7.13'
+
+homepage = 'http://bioinf.uni-greifswald.de/augustus/'
+description = "AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences"
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://bioinf.uni-greifswald.de/augustus/binaries/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['AUGUSTUS-%(version)s_fix-hardcoding.patch']
```

Comment:
We have a mix of things now, there's no point in changing this unless we force 'code style' consistency via Travis, otherwise it's a constant back & forth.